public/css/login-registration.css

```diff
@@ -43,73 +56,120 @@ hr {
 }
 
 .login {
-	background-color: transparent;
+	background-color: var(--white);
 	margin-top: 15pt;
 	font-weight: normal;
 	font-size: 14px;
-	color: #50b5dd;
-	border: 1.5px solid #50b5dd;
+	color: #619b83;
+	border: 0px;
 	text-transform: uppercase;
 	border-radius: 10px;
 	margin-bottom: 10pt;
-	padding: 8px 20px;
+	padding: 8px 24px;
 }
 
 .login:hover {
-	background-color: #50b5dd;
+	background-color: var(--green);
 	color: white;
 	cursor: pointer;
 	transition: 0.3s ease;
 }
 
-/* FORM */
+/* TITLE & SUBTITLE */
+.background h1 {
+	margin-top: 0px;
+	margin-bottom: 2px;
+}
 
+.background h2 {
+	margin-top: 4px;
+	margin-bottom: 48px;
+
+	font-size: 30px;
+}
+
+/* FORM */
 .ideahacks-form label {
-	margin-top: 10pt;
-	float: left;
+	display: block;
+	color: var(--black);
+	font-weight: bold;
+
+	margin-top: 12px;
+	margin-bottom: 4px;
+
+	margin-left: calc(10% + 12px);
+
+	text-align: left !important;
 }
 
 .ideahacks-form input,
 .ideahacks-form input:focus,
 .ideahacks-form input[readonly] {
```

Comment:
Can you add `font-weight: normal` to these rules? Having the input text be bolded looks a bit too chunky